repo/packages/N/nexus/4/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "packagingVersion": "3.0",
+  "name": "nexus",
+  "version": "3.2.1",
```

Comment:
@whyjustin I would prefer a separate PR as then I can start working with my demo. Would be great if you could keep the config parameter as it makes it easy to expose nexus via the public agents in demo environments.